fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -257,6 +257,166 @@ sebasedBiasFieldMNI="$SubjectFolder/$AtlasSpaceFolder/Results/$NameOffMRI/${Name
+                                                                         #   thickness and 10% of the cortical surface will be completely misaligned by a full cortical thickness. 
+                                                                         #   At a future time, we may be able to add support for fieldmap-less distortion correction. At this time, 
+                                                                         #   however, despite ongoing efforts, this problem is unsolved and no extant approach has been successfully 
+                                                                         #   shown to demonstrate clear improvement according to the accuracy standards of HCP-Style data analysis when 
+                                                                         #   compared to gold-standard fieldmap-based correction.
+
+
+# ------------------------------------------------------------------------------
+#  Compliance check
+# ------------------------------------------------------------------------------
+
+ProcessingMode=`opts_GetOpt1 "--processing-mode" $@`
+ProcessingMode=`opts_DefaultOpt $ProcessingMode "HCPStyleData"`
+Compliance="HCPStyleData"
+ComplianceMsg=""
+ComplianceWarn=""
+
+# -- No distortion correction method
+
+if [ "${DistortionCorrection}" = 'NONE' ]; then
+  ComplianceMsg+=" --dcmethod=NONE"
+  Compliance="LegacyStyleData"
+  log_Warn "The fMRIVolume pipeline is being run without appropriate distortion correction"
+  log_Warn "  of the fMRI image. This is NOT RECOMMENDED under normal circumstances. We will "
+  log_Warn "  attempt 6 DOF FreeSurfer BBR registration of the distorted fMRI to the undistorted"
+  log_Warn "  T1w image. Distorted portions of the fMRI data will not align with the cortical ribbon."
+  log_Warn "  In HCP data 30% of the cortical surface will be misaligned by at least half cortical "
+  log_Warn "  thickness and 10% of the cortical surface will be completely misaligned by a full "
+  log_Warn "  cortical thickness. At a future time, we may be able to add support for fieldmap-less "
+  log_Warn "  distortion correction. At this time, however, despite ongoing efforts, this problem is"
+  log_Warn "  unsolved and no extant approach has been successfully shown to demonstrate clear "
+  log_Warn "  improvement according to the accuracy standards of HCP-Style data analysis when compared"
+  log_Warn "  to gold-standard fieldmap-based correction."
+fi
+
+# -- Slice timing correction
+
+if [ "${DoSliceTimeCorrection}" = 'TRUE' ]; then
+  ComplianceMsg+=" --doslicetime=TRUE --slicetimerparams=${SliceTimerCorrectionParameters}"
+  Compliance="LegacyStyleData"
+  log_Warn "This LegacyStyleData option of slice timing correction is performed before motion"
+  log_Warn "  correction (as is typically done in legacy-style brain imaging) and thus assumes that the"
```

Comment:
Keep the `log_Warn` message in sync with the changes made to the comments above.

Reply:
done